src/Components/AddToDo.jsx

```diff
@@ -1,19 +1,25 @@
+import { observer } from "mobx-react";
 import React, { useState } from "react";
 import "./styles.css";
 
-const AddToDo = () => {
-  const [list, setList] = useState([]);
+const AddToDo = observer (({myProp}) => {
+  // const [list, setList] = useState([]);
   const [todo, setTodo] = useState();
 
   const add = () => {
```

Comment:
add what?

Reply:
This was an add todo function. changes add to addTodo